core/src/main/java/org/mskcc/cbio/portal/servlet/IGVLinkingJSON.java

```diff
@@ -32,31 +32,41 @@
 
 package org.mskcc.cbio.portal.servlet;
 
-import org.json.simple.JSONObject;
-import org.mskcc.cbio.portal.util.IGVLinking;
-
-import org.json.simple.JSONValue;
-
 import java.io.IOException;
 import java.io.PrintWriter;
+import java.net.URLEncoder;
+import java.util.ArrayList;
+import java.util.List;
+
 import javax.servlet.ServletException;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
+import org.json.simple.JSONObject;
```

Comment:
Please use Jackson instead of simple json. We are going to remove simple json from our project.